packages/xod-client-browser/webpack.config.js

```diff
@@ -89,7 +89,7 @@ module.exports = {
   plugins: [
     new webpack.NoErrorsPlugin(),
     new CopyWebpackPlugin([
-      { from: pkgpath('src/index.html') },
+      { from: pkgpath('../xod-client/src/core/assets/index.html') },
```

Comment:
The `..` could lead to bad things if we’ll reorganize packages some day. Better use:

```javascript
pkgpath('node_modules/xod-client/src/core/assets/index.html')
```